Source/Extensions/Blazorise.DataGrid/_DataGridRowEdit.razor

```diff
@@ -92,14 +92,14 @@ else if ( EditMode == DataGridEditMode.Inline )
         {
             @if ( column.ColumnType == DataGridColumnType.Command )
             {
-                @if ( ParentDataGrid?.Editable == true )
+                @if ( ParentDataGrid?.ShouldRenderCommands() == true )

```

Comment:
I don't ParentDataGrid will ever be null, so having a nullable check is not needed here.

Reply:
Agreed. I was just maintaining existing code logic. That happens in more places.

Reply:
I can clean it on other places :)

Reply:
Yea, sure there might even be a very very micro performance gain there.